foreman.fc

```diff
@@ -16,21 +16,27 @@
 # this program. If not, see http://www.gnu.org/licenses/.
 #
 
-# Foreman file contexts
+# Foreman and Ruby on Rails
 
-/etc/foreman(/.*)?                      gen_context(system_u:object_r:foreman_config_t,s0)
-/etc/puppet/node.rb                 --  gen_context(system_u:object_r:foreman_enc_t, s0)
-/etc/puppetlabs/puppet/node.rb      --  gen_context(system_u:object_r:foreman_enc_t, s0)
+/usr/lib/systemd/system/foreman.*               --  gen_context(system_u:object_r:foreman_rails_unit_file_t,s0)
+/etc/systemd/system/foreman.*                   --  gen_context(system_u:object_r:foreman_rails_unit_file_t,s0)
 
-/var/lib/foreman(/.*)?                  gen_context(system_u:object_r:foreman_lib_t,s0)
-/var/lib/foreman/db/(.*.sqlite3)?       gen_context(system_u:object_r:foreman_db_t,s0)
+/etc/foreman(/.*)?                                  gen_context(system_u:object_r:foreman_config_t,s0)
+/etc/puppet/node.rb                             --  gen_context(system_u:object_r:foreman_enc_t, s0)
+/etc/puppetlabs/puppet/node.rb                  --  gen_context(system_u:object_r:foreman_enc_t, s0)
 
-/var/log/foreman(/.*)?                  gen_context(system_u:object_r:foreman_log_t,s0)
+/var/lib/foreman(/.*)?                              gen_context(system_u:object_r:foreman_lib_t,s0)
+/var/lib/foreman/db/(.*.sqlite3)?                   gen_context(system_u:object_r:foreman_db_t,s0)
```

Comment:
Since you dropped the other sqlite code, this can also go if you want to clean it up completely.

Reply:
Hmmm right but one thing comes to my mind. We do actually use sqlite3 during build process, but if my understanding is correct the process is not yet confined therefore even if SELinux is turned on on the build machine it should work fine. Removed.

Reply:
> We do actually use sqlite3 during build process

Nope, that was changed. Since 2.1 we use nulldb during builds